sdk/turing/router/config/router_ensembler_config.py

```diff
@@ -487,14 +509,20 @@ def to_open_api(self) -> OpenApiModel:
 @dataclass
 class StandardRouterEnsemblerConfig(RouterEnsemblerConfig):
     def __init__(
-        self, experiment_mappings: List[Dict[str, str]], fallback_response_route_id: str
+        self,
+        fallback_response_route_id: str,
+        experiment_mappings: List[Dict[str, str]] = None,
+        route_name_path: str = "",
     ):
         """
         Method to create a new standard ensembler
 
-        :param experiment_mappings: configured mappings between routes and treatments
+        :param route_name_path: configured routh name path that points to the route name within a given treatment config
+                :param experiment_mappings: configured mappings between routes and treatments
+        :param fallback_response_route_id: configured final response route to be used as a fallback
         """
-        self.experiment_mappings = experiment_mappings
+        self.route_name_path = route_name_path
+        self.experiment_mappings = experiment_mappings if experiment_mappings else []
```

Comment:
Since `route_name_path` and `experiment_mappings` are optional fields, we'll need to handle such edge cases. when both are empty, we'll be letting the API return an error.

Reply:
Why not just configure the default `[]` value in the parameters?
```
experiment_mappings: List[Dict[str, str]] = [],
```

If `None` is configured, typically the typing used should be `Optional`, rather than a `List` in this case.

Reply:
> If None is configured, typically the typing used should be Optional, rather than a List in this case

Agree with this - Optional is more appropriate.

Reply:
> Why not just configure the default [] value in the parameters?

I believe there's an issue with using mutable default arguments in Python, which can potentially lead to unexpected behaviour - this causes warnings to pop on certain IDEs:
https://medium.com/geekculture/do-not-use-objects-as-default-arguments-in-python-1c940212db2e

Thanks for the reminder on `Optional`, I'll add that in to the type hint.